setup.cfg

```diff
@@ -3,3 +3,9 @@ max-line-length: 99
 
 [flake8]
 max-line-length: 99
+
+[tool:pytest]
+DJANGO_SETTINGS_MODULE = api.data_refinery_api.settings
+django_find_project = false
+python_paths = api
+python_files = api/data_refinery_api/tests.py common/**/test_*.py
```

Comment:
Also this all seems to be duplicated in .devcontainer/pytest.cfg

Reply:
good catch! forgot to remove this file.